libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,35 +7,36 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
-      Search <mat-icon>search</mat-icon>
+    <button mat-icon-button matSuffix aria-label="search icon">
```

Comment:
Why would a screen reader need to know this is an icon? What would be a better label?